.gitlab-ci.yml

```diff
@@ -24,7 +22,7 @@ test:
     script:
       - mkdir $JULIA_DEPOT_PATH # Pkg.jl#325
       - julia -e 'using InteractiveUtils; versioninfo()'
-      - julia --project -e 'using Pkg; Pkg.pkg"add Makie#master StatsMakie#master MakieGallery#master GLMakie#master; gc; test"'
+      - julia --project -e 'using Pkg; Pkg.pkg"test"'
```

Comment:
We should definitely add MakieGallery and StatsMakie master at least, no?

Reply:
No we definitely should stop with that... If MakieGallery has examples that rely on a certain version of StatsMakie, it should restrict it to that version

Reply:
should we not also test fixes of AbstractPlotting made to StatsMakie?  That way, we at least know that the master versions work together...also, fixes to MakieGallery won't propagate here for some time.  I don't really want to tag a release to fix a typo...

Reply:
We should test StatsMakie#master in StatsMakie ;) 

> I don't really want to tag a release to fix a typo...

Why not?

Reply:
well, correction - I don't want to tag a release to see if my typo fix works :D